static/src/javascripts/projects/common/modules/identity/account-profile.js

```diff
@@ -96,8 +104,58 @@ define([
         }
     };
 
+    accountProfile.prototype.avatarUploadByGAE = function (avatarForm) {
```

Comment:
Okay, yes would be good to know. I'm slightly reluctant to rewrite the Identity code to use reqwest unless absolutely necessary.